nautobot/utilities/choices.py

```diff
@@ -153,3 +153,23 @@ class ButtonColorChoices(ChoiceSet):
         (YELLOW, "Yellow"),
         (BLACK, "Black"),
     )
+
+
+class ButtonActionColorChoice(ChoiceSet):
+    """
+    Map standard button actions to Bootstrap color classes.
+    """
+
+    DEFAULT = "default"
+    ADD = "success"
+    EDIT = "warning"
+    IMPORT = "primary"
+    EXPORT = "default"
```

Comment:
Elsewhere in the UI, `EXPORT` buttons seem to generally be styled as `success`.

Also, just for completeness, how about adding:

```python
CANCEL = "default"
CLONE = "success"
CONFIGURE = "default"
INFO = "info"
DELETE = "danger"
SUBMIT = "primary"
```

In the long term it would be great if we could change all of our button definitions throughout the UI to use this ChoiceSet, as it will make any future UI migration simpler.